tests/test_configs/config.yaml

```diff
@@ -0,0 +1,153 @@
+imaging_specs:
+  local_storage_directory: .
+  external_storage_directory: .
+  subject_id: sim_test
+  tile_prefix: tile
+  tile_overlap_x_percent: 15
+  tile_overlap_y_percent: 10
+  z_step_size_um: 1
+  volume_x_um: 10615.616
+  volume_y_um: 7958.7
+  volume_z_um: 100
+  laser_wavelengths:
+  - 561  # test comment!
```

Comment:
Just for my own learning. Why does 561 need a hyphen here but not other inputs?

Reply:
Oh; this is one of two valid formats for documenting lists. What you see here is also equivalent to:
````yaml

laser_wavelengths: [561]
````
Note from [Wikipedia](https://en.wikipedia.org/wiki/YAML#Basic_components)